modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container to restrict specific container activities and enhance security.
+
+[INFO]
+====
+To run as a non-root user inside the container, your container must first add the `subuid` and `subgid` with the latest shadow-utils.
+====
+
+.Procedure
+
+. Use the public https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile] image based on Red Hat UBI.
+
+. Force run the container as the “build” user to prevent it from being “randomed” which will not have the setup needed to run as non-root.
```

Comment:
```suggestion
. Force the container to run as the `build` user. This ensures that the container setup required to run as a non-root user:
+
[source,terminal,subs="attributes+"]
----
<Harsh, can we provide an example command here?>
----
```
Or simply
```suggestion
. Run the container as the `build` user:
+
[source,terminal,subs="attributes+"]
----
<Harsh, can we provide an example command here?>
----
```

Reply:
In general, use straight quotes and backticks. Avoid "curly quotes."

Reply:
Need SME input here.

Reply:
@chmouel: What is the command to "run the container as the `build` user?